eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  SuppressionFilePath: 'eng/CredScanSuppression.json'
+  BaselineFilePath: ''
+  SourceDirectory: $(Build.SourcesDirectory)
+
+steps:
+- pwsh: |
+    New-Item -Path "${{ parameters.SourceDirectory }}" -Name "credscan.tsv"
```

Comment:
Do we need to create this file, or with the Set-Content or Add-Content below already handle that?